src/slack/test/actions.js

```diff
@@ -1,11 +1,15 @@
 import {NEW_ORDER_STATUS} from '../../sheets/constants'
-import {ORDER_COUNTRY_ACTIONS, ORDER_OFFICE_ACTIONS, ORDER_TYPE_ACTIONS} from './constants'
+import {ORDER_COUNTRY_ACTIONS, ORDER_OFFICE_ACTIONS, ORDER_TYPE_ACTIONS, DELIVERY_PLACE_ACTIONS} from './constants'
 
 export const getOrderActions = (order) => {
   if (!order.country) {
     return ORDER_COUNTRY_ACTIONS
   }
 
+  if (order.isHome === null) {
```

Comment:
@rtrembecky Why do we have an explicit null check here but a boolean check for the country and office respectively?

Reply:
because country and office are strings, so a boolean check is enough. however, `isHome` is a boolean, so for existence check we need explicit null check

Reply:
`isHome` is **boolean**, `country` and `office` are **string**. [_Initially all of them are **null** value_]

To check if the **(boolean)value** from the user was assigned to `isHome`, I checked if it is **null** or not.

For `country` and `office` also we can check if it is **null** or not, but **boolean check** also works for them properly in javascript.

Reply:
Ok, it makes sense, but its a bit ugly. A properly set up strict linter would not allow to do boolean checks on string like this, as its not good for readibility / can easily lead to human errors.

Reply:
I have never used/seen an ESLint rule to prevent doing `!string` in a project 😄  but I understand it's an interesting safety measure. you meant something like this one, right? https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md